src/equations/numerical_fluxes.jl

```diff
@@ -261,6 +261,23 @@ function Base.show(io::IO, d::DissipationLaxFriedrichsEntropyVariables)
     print(io, "DissipationLaxFriedrichsEntropyVariables(", d.max_abs_speed, ")")
 end
 
+@doc raw"""
+    MatrixDissipationWintersEtal()
+
+Creates the Roe-like entropy stable matrix dissipation operator from Winters et al (2017). This operator
+must be used together with an entropy-conservative two-point flux function (e.g., `flux_ec`) to yield 
+an entropy-stable surface flux. The surface flux function can be initialized as:
+```julia
+flux_es = FluxPlusDissipation(flux_ec, MatrixDissipationWintersEtal())
+```
+This implementation is adapted from the [Atum.jl library](https://github.com/mwarusz/Atum.jl/blob/c7ed44f2b7972ac726ef345da7b98b0bda60e2a3/src/balancelaws/euler.jl#L198).
+For the derivation of the matrix dissipation operator, see:
+- A. Winters, D. Derigs, G. Gassner, S. Walch, A uniquely defined entropy stable matrix dissipation operator 
+  for high Mach number ideal MHD and compressible Euler simulations (2024). Journal of Computational Physics.
+  [DOI: 10.1016/j.jcp.2016.12.006](https://doi.org/10.1016/j.jcp.2016.12.006).
+"""
+struct MatrixDissipationWintersEtal end
```

Comment:
Consistency with the other dissipation operators would suggest to use something like `DissipationMatrixWintersEtal`. @sloede?

Reply:
Right. Unless "matrix dissipation" is really the "noun" we want here, because there are multiple "matrix dissipation" types. If in doubt, I'd go with @ranocha's suggestion, but I'll leave it up to you

Reply:
Thanks, that makes sense. I started with `DissipationWintersEtal` but thought that might be too general - will update the PR.